packages/chart/src/Pie.ts

```diff
@@ -1,8 +1,11 @@
 import { I2DChart, ITooltip } from "@hpcc-js/api";
 import { InputField, SVGWidget, Utility } from "@hpcc-js/common";
+import { degreesToRadians, normalizeRadians } from "@hpcc-js/util";
 import { interpolate as d3Interpolate } from "d3-interpolate";
 import { select as d3Select } from "d3-selection";
 import { arc as d3Arc, pie as d3Pie } from "d3-shape";
+import { transition as d3Transition } from "d3-transition";
+d3Select.prototype.transition = d3Transition;
```

Comment:
This does not look correct to me?

Reply:
Without it, the linter complains about transition not existing on `Selection`

Reply:
Change to: 
```
import "d3-transition";
```